src/views/preview/presentation.jsx

```diff
@@ -489,7 +490,7 @@ const PreviewPresentation = ({
                                 <div className="comments-container">
                                     <FlexRow className="comments-header">
                                         <h4><FormattedMessage id="project.comments.header" /></h4>
-                                        {userOwnsProject ? (
+                                        {canToggleComments ? (
```

Comment:
There's a lint error above because this was the only place in the file `userOwnsProject` was used.